scripts/ci/index_ref_doc.py

```diff
@@ -57,7 +58,11 @@ def test(self):
                     os.mkdir(ref_doc_out_dir)
                 script_args = [sys.executable, REF_GEN_SCRIPT, '--extension-file', ext_file, '--output-dir',
                                ref_doc_out_dir]
-                check_call(script_args)
+                try:
+                    check_call(script_args)
+                except CalledProcessError as e:
```

Comment:
why speific this error? what about other errors?

Reply:
It raises this kind of error when it fails according to the doc. It's not recommended to catch a too-broad exception.